public/css/main.stylesheet.css

```diff
@@ -748,6 +767,15 @@ div.icon img {
   outline: none;
 }
 
+.moduleProperty input {
+  background-color: transparent;
+  color: #fff;
+  margin-top: 7px;
+  outline: none;
+  border: 1px solid white;
+  padding: 0 5px;
```

Comment:
can we bring the `width: 100%` to here instead of [keeping it separately](https://github.com/CircuitVerse/CircuitVerse/pull/1584/files#diff-d4ca5df21e37487be60366d7f1d66054R758)?

Reply:
sure!

Reply:
y is already 5px?

Reply:
As of now, Y is 0px so it'll be padding: 5px 5px;